docs/documentation/backend/command_line_arguments.md

```diff
@@ -4,22 +4,22 @@
 | `-fn` `--file-name` | `String`  | Name and path to the route file (e.g. GPX or KML file). Required Argument. |
 
-### Optional Arguments
+### Settings: Optional Arguments
 
 | Name                    | arguments     | Description                                                                                                                                                                                                                                                                                               |
 |-------------------------|---------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | `-v` `--velocity`       | `Float`       | Speed in km/h on which the calculation is based, default 3.75 km/h.                                                                                                                                                                                                                                       |
 | `-s` `--map-scaling`    | `Integer`     | Scaling of the created map (e.g. 10000 for scaling of 1:10'000). If not specified, the scaling will be automatically chosen such that the path can be printed onto a single A4 paper. The scaling gets chosen out of a list of common map scaling: 1:10'000, 1:25'000, 1:50'000, 1:100'000, or 1:200'000. |
 | `-t` `--departure-time` | ISO-timestamp | Departure date in ISO-format, i.g. 2011-11-04T00:05:23. Default 2021-08-16T09:00:00.                                                                                                                                                                                                                      |
 | `-n` `--creator-name`   | `String`      | The name of the creator of this walk-table. Default is just an empty string.                                                                                                                                                                                                                              |
-| `--log-level`           | `Integer`     | Log Level (see https://docs.python.org/3/library/logging.html#levels). Default: WARNING                                                                                                                                                                                                                   |
+| `--log-level`           | `Integer`     | Log Level. Default: WARNING                                                                                                                                                                                                                                                                               |
```

Comment:
The log level is also configured using the environment file. Where is the difference? Can we drop this argument? Or should we keep it to support the python CLI script? May be this will be helpful to perform unit tests of the backend? What do you think? @maede97

Reply:
I like it more with the env file. I think we should go away from CLI.